core/modules/file.py

```diff
@@ -84,9 +86,10 @@ def unserialize(self, skel, name):
 
 
 def thumbnailer(fileSkel, existingFiles, params):
-    blob = bucket.get_blob("%s/source/%s" % (fileSkel["dlkey"], fileSkel["name"]))
+    orgifileName = fileSkel["name"].replace("&#040;", "(").replace("&#041;", ")").replace("&#061;", "=")
```

Comment:
We must replace here to create derives of an image with one of these characters, because the filename is stored in a StringBone and this will be escaped

Reply:
Yes we can make that I copy the code from here:
https://github.com/viur-framework/viur-core/blob/312d24b96b880e01ee3902043795b1a39633a79c/core/utils.py#L149